solidity/contracts/vault/TBTCVault.sol

```diff
@@ -74,68 +74,92 @@ contract TBTCVault is IVault, Ownable, TBTCOptimisticMinting {
         tbtcToken = _tbtcToken;
     }
 
-    /// @notice Transfers the given `amount` of the Bank balance from caller
-    ///         to TBTC Vault, and mints `amount` of TBTC to the caller.
-    /// @dev TBTC Vault must have an allowance for caller's balance in the Bank
-    ///      for at least `amount`.
+    /// @notice Mints the given `amount` of TBTC to the caller previously
+    ///         transferring `amount / SATOSHI_MULTIPLIER` of the Bank balance
+    ///         from caller to TBTC Vault. If `amount` is not divisible by
+    ///         SATOSHI_MULTIPLIER, the remainder is left on the caller's
+    ///         Bank balance.
+    /// @dev TBTC Vault must have an allowance for caller's balance in the
+    ///      Bank for at least `amount / SATOSHI_MULTIPLIER`.
     /// @param amount Amount of TBTC to mint.
     function mint(uint256 amount) external {
+        (
+            uint256 strippedAmount,
+            ,
+            uint256 balanceToTransfer
+        ) = amountToSatoshis(amount);
```

Comment:
Shouldn't we use names returned by `amountToSatoshis `? Here and everywhere else where `amountToSatoshis` is used?

Reply:
We should, my fault. See db846c182e000762b8a577a8935f29517072a0f7, please.